src/lib/core/src/chunks/chunk_receiver.rs

```diff
@@ -0,0 +1,46 @@
+use dashmap::{DashMap, DashSet};
+use ferrumc_world::chunk_format::Chunk;
+use tokio::time::Instant;
+
+const VIEW_DISTANCE: i32 = 8;
```

Comment:
the view distance is hard-coded, add it to and pull from config, if possible have some reference to the player so in future we can grab their client-side view distance: https://minecraft.wiki/w/Minecraft_Wiki:Projects/wiki.vg_merge/Protocol#Client_Information_(configuration) so we arent sending them chunks they immediately discard if their distance is smaller than config distance